client/verta/verta/_deployment/endpoint.py

```diff
@@ -470,3 +476,48 @@ def get_update_status(self):
         response = _utils.make_request("GET", url, self._conn)
         _utils.raise_for_http_error(response)
         return response.json()
+
+    def download_manifest(
+            self, download_to_path, name, strategy,
+            resources=None, autoscaling=None, env_vars=None):
+        """
+        Downloads this Endpoint's Kubernetes manifest YAML.
+
+        Parameters
+        ----------
+        download_to_path : str
+            Path to download manifest YAML to.
+        name : str
+            Name of the Endpoint.
```

Comment:
@conradoverta Is this an accurate description for `metadata.name`?

Reply:
yep